crates/test_utils/Cargo.toml

```diff
@@ -26,6 +27,7 @@ actix-http = "3.3.1"
 awc = { version = "3.1.1", features = ["rustls"] }
 derive_deref = "1.1.1"
 rand = "0.8.5"
+reqwest = { version = "0.11.18", features = ["json", "native-tls", "gzip", "multipart"] }
```

Comment:
Do we need all of these features to be enabled?

Reply:
No , we only need `native-tls` , i will remove other features